src/dash-table/derived/cell/operations.tsx

```diff
@@ -2,31 +2,38 @@ import * as R from 'ramda';
 import React from 'react';
 
 import { memoizeOneFactory } from 'core/memoizer';
+import { clearSelection } from 'dash-table/utils/actions';
 
 import {
     Data,
     Datum,
     SetProps,
     RowSelection,
-    ActiveCell,
     Indices
 } from 'dash-table/components/Table/props';
 
-function deleteRow(idx: number, activeCell: ActiveCell, data: Data, selectedRows: number[]) {
+function deleteRow(idx: number, data: Data, selectedRows: number[]) {
     const newProps: any = {
-        data: R.remove(idx, 1, data)
+        data: R.remove(idx, 1, data),
+        // We could try to adjust selection, but there are lots of edge cases
+        ...clearSelection
```

Comment:
previously we left `selected_cells` intact when deleting a row, and only cleared `active_cell` if it was actually deleted. But that has some problems, and for simplicity and symmetry with [`deleteColumns`](https://github.com/plotly/dash-table/pull/412/commits/572cdf13ae78eac83fdc79ef5db5bc67468d8cc7#diff-4102a065e79334d16000051ec35189f7R46) I decided to just clear the selection here too.